prac_05/emails.py

```diff
@@ -0,0 +1,26 @@
+def main():
+
+    email_to_name = {}
+
+    email = input('Email: ')
+
+    while email != '':
+        full_name = return_name_from_email(email)
+        confirm_full_name = input(f'Is your name {full_name}? (Y/n) ')
+        if confirm_full_name == 'n':
+            full_name = input("Name: ").title()
+        email_to_name[email] = full_name
+        email = input('Email: ')
```

Comment:
Same as above comment, would be good to use double quotes for consistency as it's longer than a single character